### PR TITLE
[12.0][IMP] sale_financial_risk: extrapolate risk_exception_msg method in order to get it overridable

### PR DESCRIPTION
**Pull request by tafaRU**

### CODE REVIEW

File: sale_financial_risk/models/sale.py [hunk 1]
```diff
@@ -8,30 +8,34 @@
...
-                    (partner.risk_total + risk_amount) >
-                    partner.credit_limit):
-                exception_msg = _(
-                    "This sale order exceeds the financial risk.\n")
+            exception_msg = self.risk_exception_msg()
             if exception_msg:
                 return self.env['partner.risk.exceeded.wiz'].create({
                     'exception_msg': exception_msg,
-                    'partner_id': partner.id,
+                    'partner_id': self.partner_id.commercial_partner_id.id,
```
Comment (pedrobaeza) [inline sale_financial_risk/models/sale.py:38]: Why this change?

Comment (tafaRU) [reply sale_financial_risk/models/sale.py:38]: just do not repeat `partner` equal to `self.partner_id.commercial_partner_id`

Comment (pedrobaeza) [reply sale_financial_risk/models/sale.py:38]: OK, I see.

**Review by pedrobaeza — APPROVED** [review]

**Comment by tafaRU** [comment]
@pedrobaeza @carlosdauden could we merge this? Otherwise I'll add https://github.com/OCA/credit-control/issues/82 here. Please let me know what you prefer.

**Comment by pedrobaeza** [comment]
OK for me.

**Comment by tafaRU** [comment]
> OK for me.

Ok @pedrobaeza, I proceed with the merge then.

**Review by carlosdauden — APPROVED** [review]

**Comment by tafaRU** [comment]
@carlosdauden thanks for the approve. Sorry for not waiting longer but I thought you were on vacation because there was no answer on https://github.com/OCA/credit-control/issues/82 :wink:
